### ISSUE TITLE
Improve lore2sphinx buildbot results for `projects/core/howto/options.xhtml`

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4876]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4876|
|Type|enhancement|
|Created|2011-02-11 05:42:07Z|
|Branch|https://github.com/twisted/twisted/tree/options-sphinx-4876|

like [#4566](https://github.com/twisted/twisted/issues/4566), except for the options howtos

<details><summary>Searchable metadata</summary>

```
trac-id__4876 4876
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__Lore_to_Sphinx Lore to Sphinx
branch__branches_options_sphinx_4876 branches/options-sphinx-4876
branch_author__exarkun__screwtape__jdb exarkun, screwtape, jdb
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1297402927000000 1297402927000000
changetime__1300075329000000 1300075329000000
version__None None
owner__lvh lvh

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [30684]) Branching to 'options-sphinx-4876'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [30685]) Merge the options parts of the impossibly huge [[#4568](https://github.com/twisted/twisted/issues/4568)](https://github.com/twisted/twisted/issues/4568) branch

refs #4568
refs [#4876](https://github.com/twisted/twisted/issues/4876)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun commented|
|-|-|

(In [30686]) Branching to 'telnet-sphinx-4876'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun commented|
|-|-|

Oops, typo.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)|@khorn set owner to @khorn|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_9"></a>|@khorn set owner to @lvh|
|-|-|

Yay. Merge.

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_10"></a>|@lvh set status to `closed`|
|-|-|

(In [31001]) Merge options.xhtml changes from options-sphinx-4876.


Author: screwtape, jdb
Reviewer: khorn
Fixes: [#4876](https://github.com/twisted/twisted/issues/4876)
  
"Makes Sphinx looks gooder." -- JP
